src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx

```diff
@@ -160,14 +162,16 @@ function WorkspaceExpensifyCardListPage({route}: WorkspaceExpensifyCardListPageP
             >
                 {!shouldUseNarrowLayout && getHeaderButtons()}
             </HeaderWithBackButton>
-
             {shouldUseNarrowLayout && <View style={[styles.pl5, styles.pr5]}>{getHeaderButtons()}</View>}
-
-            <FlatList
-                data={sortedCards}
-                renderItem={renderItem}
-                ListHeaderComponent={WorkspaceCardListHeader}
-            />
+            {!isEmptyObject(cardsList) ? (
```

Comment:
@koko57 Please help to explain this line 

As I understand, we display EmptyCardView if cardList í empty

Reply:
yes

Reply:
yes, I did it probably bc I didn't wanted to remove mockedCard list in this PR. I'll tell Vicky (she's integrating BE with the cardList), to correct this condition

Reply:
yes, she did it in her PR https://github.com/Expensify/App/pull/47064/files